### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by asleepysheepy**
Update's the README for version 2.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,53 +73,145 @@ If you want to invite the public Winnie_Bot account to your server, go [here](ht
...
+
+## Core Team
+
+| Dawn E. Collett | Jason E. Gillikin | Katie Macke |
+|---|---|---|
+| [![Dawn's Github](https://avatars1.githubusercontent.com/u/28942094?s=100&v=4)](https://github.com/lisushka) | [![Jason's Github](https://avatars2.githubusercontent.com/u/7763031?s=100&v=4)](https://github.com/jegillikin) | [![Katie's Github](https://avatars1.githubusercontent.com/u/12132647?s=100&v=4)](https://github.com/asleepysheepy) |
 
 ## License
 
-This project is licensed under the GNU General Public License, v3.0 - see the [LICENSE](LICENSE) file for details.  In particular, if you incorporate Winnie's source code into another project, you must release the source code of that project.
```
Comment (lisushka) [inline README.md:84]: Can we add the line about needing to release your source code under GPL back in?

File: README.md [hunk 2]
```diff
@@ -44,53 +73,145 @@ If you want to invite the public Winnie_Bot account to your server, go [here](ht
...
 
 Please report bugs by opening an [issue](https://github.com/lisushka/Winnie_Bot/issues) on GitHub.
 
-## Contributing to Winnie
+### Translations
 
-We welcome all contributors to Winnie.  Your pull requests will be reviewed by the authors prior to merging.  Please document your code, and play nicely with other contributors.
+#### Adding a new langauge
+
+If you want to help tanslate Winnie into a language she does not currently support, please join our [community Discord](https://discord.gg/mvZZMhK) to get in touch with the team.
```
Comment (lisushka) [inline README.md:114]: Mend typo in `translate`

**Review by lisushka — APPROVED** [review]
I'm happy with this as an initial cut, with the listed changes.  Once it's in, we can improve it.

File: README.md (hunk 1 above)
Comment (asleepysheepy) [reply README.md:84]: whoops, didn't even know that was there

File: README.md (hunk 2 above)
Comment (asleepysheepy) [reply README.md:114]: 🤦‍♀️

**Pull request merged**